### ISSUE TITLE
rename?

### ISSUE THREAD

**Issue by plebhash** [demand-open-source/coinbase-monitor#3]
when I first tried to use this tool, I thought it was giving me information about the coinbase of the network tip

admitedly, that was before actually reading the source code 🙈

then I realized that it is actually leveraging Sv2 TDP to monitor for coinbases of **Sv2 jobs**

suggestion for names that would be better IMO:
- `sv2-tdp-coinbase-monitor`
- `sv2-template-coinbase-monitor`

maybe there's other better options idk

the main point is that I think this tool would be better understood if it explicitly mentions Sv2 in the naming

**Comment by Fi3** [comment]
yep we can change it no problem, but keep in mind that this is mostly an example of how to use the easy-sv2 library. Of course if someone find it useful and we want to further develop it I'm open to it

**Comment by plebhash** [comment]
the suggestions above are heavily based on the assumption that this tool will only be used to monitor TDP

but that doesn't have to necessarily be the case, we could also potentially expand functionality

it could also be useful to connect this tool to:
- JDC (to monitor coinbase under JD Protocol)
- Pool (to monitor coinbase under Mining Protocol)
- Proxy (to monitor coinbase under Mining Protocol)

in that case, a better name would be `sv2-coinbase-monitor`
